### PR TITLE
Generate .cabal file from package.yaml

### PR DESCRIPTION
**Pull request by mjgpy3**
Our release action needs this.

### CODE REVIEW

**Comment by mjgpy3** [comment]
I don't know if I did this correctly or if there's an automated way to build the package.yaml from the cabal file.

File: package.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+name: yesod-routes-flow
+version: 3.0.0.1
+category: Web
+author: Freckle Engineering
+maintainer: engineering@freckle.com
+copyright: 2021 Freckle Education
```
Comment (pbrisbin) [inline package.yaml:6]: I think this supposed to be Renaissance Learning, but I'm not sure. Check some of our other OSS.

**Review by pbrisbin — APPROVED** [review]
LGTM, I used to use [this thing](https://github.com/yamadapc/hpack-convert) back in the day when hpack was just taking over, but I'm not sure if it works or is up anymore.

File: package.yaml (hunk 1 above)
Comment (mjgpy3) [reply package.yaml:6]: Oh, I think you're right!

**Pull request merged**